app/manifestV2.json

```
@@ -1,6 +1,6 @@
 {
-	"name": "Interceptor",
-	"description": "Interceptor",
+	"name": "The Interceptor",
+	"description": "Gain a clearer understanding of your Ethereum transactions with The Interceptor browser extension",
```

Offering a description suggestion

```suggestion
	"description": "Know exactly what is happening with your Ethereum transactions without putting anything at risk",
```

---

```suggestion
	"description": "Gain a clearer understanding of your Ethereum transactions before any of your assets are at risk.  Simulate a series of transactions to see what will happen before signing anything.  Get warned of common scams and mistakes before you become a victim of them.",
```